### ISSUE TITLE
Android input field over chat

### ISSUE THREAD

**Issue by bcsopi** [indywidualny/FaceSlim#134]
Hello, how would it be to implement an android system input field on the bottom of any chat page, thus making it incredibly more comfortable to write/copy/paste text, when writing messages. Just because eg. pasting links into facebook's default chat field is very much of a hassle for me now. :D

**Comment by NeroBurner** [comment]
Yes it would be awesome to actually see what I'm writing in a comment :D

**Comment by indywidualny** [comment]
It's a nice idea. I will try to add it later this month.

**Comment by kelna** [comment]
+1

**Comment by kelna** [comment]
The same would be nice for posts. The input field is too small. If I want to read again or change what I typed a few lines above, I have to pull the text down, but this can lead to the page reloading and the text being gone forever. This is very frustrating.
